implants/lib/eldritch/src/pivot.rs

```diff
@@ -61,41 +60,157 @@ impl<'v> UnpackValue<'v> for PivotLibrary {
+    fn ssh_exec<'v>(
+        this: PivotLibrary,
+        starlark_heap: &'v Heap,
+        target: String,
+        port: i32,
+        command: String,
+        username: String,
+        password: Option<String>,
+        key: Option<String>,
+        key_password: Option<String>,
+        timeout: Option<u32>,
+    ) -> anyhow::Result<Dict<'v>> {
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
+        ssh_exec_impl::ssh_exec(
+            starlark_heap,
+            target,
+            port,
+            command,
+            username,
+            password,
+            key,
+            key_password,
+            timeout,
+        )
     }
-    fn ssh_password_spray(this:  PivotLibrary, targets: Vec<String>, port: i32, credentials: Vec<String>, keys: Vec<String>, command: String, shell_path: String) ->  anyhow::Result<String> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
-        ssh_password_spray_impl::ssh_password_spray(targets, port, credentials, keys, command, shell_path)
+    fn ssh_password_spray(
+        this: PivotLibrary,
+        targets: Vec<String>,
+        port: i32,
+        credentials: Vec<String>,
+        keys: Vec<String>,
+        command: String,
+        shell_path: String,
+    ) -> anyhow::Result<String> {
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
+        ssh_password_spray_impl::ssh_password_spray(
+            targets,
+            port,
+            credentials,
+            keys,
+            command,
+            shell_path,
+        )
     }
-    fn smb_exec(this:  PivotLibrary, target: String, port: i32, username: String, password: String, hash: String, command: String) ->  anyhow::Result<String> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+    fn smb_exec(
+        this: PivotLibrary,
+        target: String,
+        port: i32,
+        username: String,
+        password: String,
+        hash: String,
+        command: String,
+    ) -> anyhow::Result<String> {
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         smb_exec_impl::smb_exec(target, port, username, password, hash, command)
-    }     
+    }
     // May want these too: PSRemoting, WMI, WinRM
-    fn port_scan<'v>(this:  PivotLibrary, starlark_heap: &'v Heap, target_cidrs: Vec<String>, ports: Vec<i32>, protocol: String, timeout:  i32) ->  anyhow::Result<Vec<Dict<'v>>> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+    fn port_scan<'v>(
```

Comment:
Not sure if this more readable normally a big fan of the multiline but with the amount of repetition not sure here.
The module files are mostly boiler plate maybe we want add an exception for this file at least for the multiline rule.

Other folks have thoughts?

Reply:
This is also an odd situation because clippy did have other issues about these functions that I'm ignoring. It really doesn't like how many params it has. I think that alongside making it multi line made it very long

Reply:
Ohhhh is that gonna cause issues? If we're not adhering to the clippy format 100%?

Reply:
discussion point